lib/rubocop/cop/rspec/spec_file_path_format.rb

```diff
@@ -57,8 +63,67 @@ def on_top_level_example_group(node)
           end
         end
 
+        # For testing and debugging
+        def self.reset_activesupport_cache!
```

Comment:
I am not crazy about this method. It feels like maybe we are memoizing on the wrong level somehow.